.gitignore

```diff
@@ -1,8 +1,10 @@
 *.iml
-.gradle
-/local.properties
-/.idea
+.gradle/
+local.properties
+.idea/
 .DS_Store
-/build
-/captures
-.externalNativeBuild
+build/
+captures/
+.externalNativeBuild/
+app/github/
```

Comment:
app/build is missing.

Reply:
This is automatically covered by `build/`.

Reply:
wow, that's kinda confusing... Maybe just restore old gitignore files...

Reply:
Alright 👍